M2/Macaulay2/m2/exports.m2

```diff
@@ -739,6 +739,7 @@ export {
 	"id",
 	"ideal",
 	"identity",
+    "identical",
```

Comment:
Does this `identical(x,y)` differ from `x===y` somehow for objects of type RRi?  I would expect it not to.

Reply:
Let's settle what x===y does first -- maybe it will end up doing what "identical" now does.